vm/classes/classes.go

```diff
@@ -15,4 +15,5 @@ const (
 	PluginClass   = "Plugin"
 	GoObjectClass = "GoObject"
 	FileClass     = "File"
+	RegexpClass   = "Regexp"
```

Comment:
Why do you name it `Regexp` instead of `Regex`? What does the `p` mean?

Reply:
I think the `p` is part of `Exp`ression. Ruby names the regular expression class `Regexp` rather than `Regex`.

Reply:
@saveriomiroddi I see, thanks for the explanation 😄